Assets/Player/PsychosisBar.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PsychosisBar : MonoBehaviour
+{
+    public Slider slider;
+    private float _timer;
+    public float currentValue;
+    public float coolDownTime;
```

Comment:
This is completely my bad as the Documentation is not updated but public variables use PascalCase, you can  make them `[SerializeField] private` instead, that uses camelCase if you are sure they don't need to be accessed outside the file